src/resources/views/layouts/_partials/_admin/_event/_tickets/ticket-group-form.blade.php

```diff
@@ -0,0 +1,44 @@
+@if ($errors->any())
+    <div class="alert alert-danger">
+        <ul>
+            @foreach ($errors->all() as $error)
+                <li>{{ $error }}</li>
+            @endforeach
+        </ul>
+    </div>
+@endif
+<div class="row">
+    <div class="mb-3 col-12">
+        {{ Form::label('ticket-group-name','Ticket group name') }}
+        {{
+            Form::text(
+                'ticket-group-name',
+                isset($ticketGroup) ? $ticketGroup->name : null,
+                [
+                    'id'=>'ticket-group-name',
+                    'class'=>'form-control'
+                ]
+            )
+        }}
+    </div>
+</div>
+
+<div class="row">
+    <div class="mb-3 col-12">
+        {{ Form::label('ticket-group-tickets','Ticket group name') }}
```

Comment:
wrong label

Reply:
Copy-paste is an evil mistress... The label now reflects the label in the ticket groups table